bin/dokku-daemon

```diff
@@ -75,7 +61,7 @@ server() {
   # start accepting connections
   log "Listening on $DOKKU_SOCK_PATH..."
 
-  socat unix-listen:"$DOKKU_SOCK_PATH",fork exec:"$0 -c",fdin=3,fdout=4
+  socat unix-listen:"$DOKKU_SOCK_PATH",fork,perm-early=0777 exec:"$0 -c",fdin=3,fdout=4
```

Comment:
Can you explain this?

Reply:
At least in the environments I tested in, I found that later versions of SystemD had issues with permissions on the created socket. By adding the permissions to `socat` directly, I was able to resolve the issue. I wasn't able to identify any negative consequences to this.